src/com/clover/remote/client/transport/websocket/WebSocketPairedCloverTransport.ts

```diff
@@ -92,12 +79,12 @@ export class WebSocketPairedCloverTransport extends WebSocketCloverTransport {
                 if (sdkMessage) {
                     if (sdk.remotemessage.Method.PAIRING_CODE == sdkMessage.getMethod()) {
                         this.logger.debug("Got PAIRING_CODE");
-                        var pcm: sdk.remotemessage.PairingCodeMessage = sdkMessage;
+                        var pcm: sdk.remotemessage.PairingCodeMessage = <sdk.remotemessage.PairingCodeMessage> sdkMessage;
```

Comment:
casting comment again...

Reply:
PairingCodeMessage is a Message, Message is not a PairingCodeMessage.

Reply:
again, I think that a PairingCodeMessage is what's passed and it's only stored in a Message, so it should be ok, but we'll continue base on the other comment rather than having the discussion twice.  Thanks.